CHANGES

```diff
@@ -1,6 +1,35 @@
 Release 1.2 (in development)
 ----------------------------
 
+.. rubric:: Enhancements
+
+* Let rauc-hawkbit-updater use the recent InstallBundle() DBus method instead of
+  legacy Install() method. [#129]
+
+.. rubric:: Bug Fixes
+
+* fixed NULL pointer dereference if build_api_url() is called for base
+  deployment URL with having GLIB_USING_SYSTEM_PRINTF defined [#115]
+* fixed compilation against musl by not including glibc-specific
+  bits/types/struct_tm.h [#123] (by Zygmunt KrynickiA)
+
+.. rubric:: Code
+
+* drop some unused variable [#126]
+
+.. rubric:: Testing
+
+* enable and fix testing for ipv6 addresses [#116]
+* enhance test output by not aborting to early on process termination [#128]
+* set proper names for python logger [#127]
+
+.. rubric:: Documentation
+
+* corrected retry_wait default value in reference [#118]
+* suggest using systemd-tmpfiles for creating and managing tmp directories
+  [#124] (by Jonas Licht)
```

Comment:
We should probably mention that this is only relevant for plain RAUC bundles.

Reply:
I've adapted the line